src/test/java/uk/gov/companieshouse/items/orders/api/mapper/CertificateItemMapperTest.java

```diff
@@ -62,9 +65,15 @@
     private static final boolean INCLUDE_DATES = true;
 
     private static final CertificateItemOptions ITEM_OPTIONS;
+    private static final CertificateItemOptions ITEM_OPTIONS_NO_DEFAULTS;
     private static final DirectorOrSecretaryDetails DIRECTOR_OR_SECRETARY_DETAILS;
     private static final RegisteredOfficeAddressDetails REGISTERED_OFFICE_ADDRESS_DETAILS;
 
+    private static final CertificateType NO_DEFAULT_CERTIFICATE_TYPE = INCORPORATION_WITH_ALL_NAME_CHANGES;
+    private static final DeliveryMethod NO_DEFAULT_DELIVERY_METHOD = POSTAL;
```

Comment:
Is there a test to check the default values are set if no values are set? These variables appear to be unused

Reply:
Updated.